### ISSUE TITLE
Error when installing on remote servers

### ISSUE THREAD

**Issue by diego04** [ranisalt/node-argon2#4]
Hi, by any chance do you know how to solve this:
I can't seem to get it to work 

```
remote:        npm ERR! argon2@0.5.1 preinstall: `git submodule update --init`
remote:        npm ERR! Exit status 128
remote:        npm ERR!
remote:        npm ERR! Failed at the argon2@0.5.1 preinstall script 'git submodule update --init'.
remote:        npm ERR! Make sure you have the latest version of node.js and npm installed.
remote:        npm ERR! If you do, this is most likely a problem with the argon2 package,
remote:        npm ERR! not with npm itself.
remote:        npm ERR! Tell the author that this fails on your system:
remote:        npm ERR!     git submodule update --init
remote:        npm ERR! You can get their info via:
remote:        npm ERR!     npm owner ls argon2
remote:        npm ERR! There is likely additional logging output above.
remote:        
```

**Comment by ranisalt** [comment]
I noticed it happens because you are not inside a git directory and I could reproduce it. I will investigate how to fix it.

**Comment by ranisalt** [comment]
It should be fixed with v0.6.0
